app/views/shared/_header.html.erb

```diff
@@ -7,12 +7,12 @@
       <div>
         <%= current_user.surname %>
       </div>
-      <%= link_to "New article", new_article_path %>
-      <%= link_to "Sign out", destroy_session_path(:user) %>
+      <%= link_to (I18n.t "articles.paths.new"), new_article_path %>
```

Comment:
I think it is better to keep one style. Sometimes you use `I18n.t '...'`, sometimes `(I18n.t '...')`

Reply:
i change on `t(` or `t`